requirements.txt

```diff
@@ -48,3 +48,4 @@ flashinfer-python~=0.2.3
 opencv-python-headless
 xgrammar==0.1.16
 backoff
+etcd3
```

Comment:
Please merge the Docker image upgrade as a separate PR first, and put the changes to `requirements.txt` in the next PR. Thanks!